streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1587,11 +1587,15 @@ public void handleStreamsRebalanceData() {
     }
 
     private void handleMissingSourceTopicsWithTimeout(final String missingTopicsDetail) {
+        // Determine the timeout: use 2 * heartbeatIntervalMs
```

Comment:
This comment just says, what the code does (-> `timeoutMs = 2 * heartbeatIntervalMs`) -- it does not add any knowledge. -- Either remove it, or say _why_ we need to use `2* heartbeatIntervalMs` as timeout